mwibutsa/Questioner-revised
# #163149432 Creates user login page

#### What does this PR do?
- adds a user login page
#### Description of Task to be completed?
- create a user login form with username, password and submit controls 
#### How should this be manually tested?
- user should type their account credentials in corresponding fields and then submit
#### Any background context you want to provide?
- this is the UI is requires the back-end to fully function
#### What are the relevant pivotal tracker stories?
#163149432
#### Screenshots
![capture](https://user-images.githubusercontent.com/36665761/51053115-6e11dc80-15e1-11e9-97ac-d620f98e8e67.PNG)
